### PR TITLE
[Docs] Added Ionic and Cordova compatibility information

### LINKED ISSUE #701: [Android][iOS][Cordova]Virtual keyboard doesn't work as an input
## 🐞 Is this a bug report or feature request? (choose one)

- Bug report

## 💻 Version of CKEditor

CKEditor 5 classic editor build 1.0.0-alpha.2

## 📋 Steps to reproduce

1. Create an editor
2. tap into the editor area
3. when the virtual keyboard appears, type on the virtual keyboard

## ✅ Expected result
typed text should appear in the editor area

## ❎ Actual result
the editor area stays empty and the value of the editor also null

## 📃 Other details that might be useful
The application runs in a native WebView (built by cordova)
after I fixed the addListener problem by #683

### PR DESCRIPTION
**Pull request by Mgsy**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Added Ionic and Cordova compatibility information. Closes #701

### CODE REVIEW

File: docs/builds/guides/frameworks/overview.md [hunk 1]
```diff
@@ -41,3 +41,13 @@ We plan to provide more integrations with time. We would like to [hear your idea
 Starting from version 11.0.0 CKEditor 5 is compatible with Electron. Using CKEditor 5 in Electron applications does not require any additional steps.
 
 Check out a [sweet screencast of CKEditor 5 with real-time collaborative editing in Electron](https://twitter.com/ckeditor/status/1016627687568363520).
+
+## Compatibility with Ionic and Cordova
+
+CKEditor 5 is compatible with Ionic framework and Cordova. However, Ionic disables `user-select` by default, what [blocks typing via the virtual keyboard](https://github.com/ckeditor/ckeditor5/issues/701). It's required to set this property to `text` on the CKEditor component. Just update your stylesheet with the following code:
+```css
```
Comment (Reinmar) [inline docs/builds/guides/frameworks/overview.md]: Missing blank line above.

File: docs/builds/guides/frameworks/overview.md [hunk 2]
```diff
@@ -41,3 +41,13 @@ We plan to provide more integrations with time. We would like to [hear your idea
 Starting from version 11.0.0 CKEditor 5 is compatible with Electron. Using CKEditor 5 in Electron applications does not require any additional steps.
 
 Check out a [sweet screencast of CKEditor 5 with real-time collaborative editing in Electron](https://twitter.com/ckeditor/status/1016627687568363520).
+
+## Compatibility with Ionic and Cordova
+
+CKEditor 5 is compatible with Ionic framework and Cordova. However, Ionic disables `user-select` by default, what [blocks typing via the virtual keyboard](https://github.com/ckeditor/ckeditor5/issues/701). It's required to set this property to `text` on the CKEditor component. Just update your stylesheet with the following code:
```
Comment (Reinmar) [inline docs/builds/guides/frameworks/overview.md]: Missing links to Ionic and Cordova.

**Comment by Reinmar** [comment]
Two minor issues. You can merge it once your resolve them.

**Comment by sicosola** [comment]
After processing in this way, I was able to enter text, but I still can't select the text I have entered.

**Pull request merged**